bin/reindex

```diff
@@ -7,6 +7,7 @@ if [ "$#" -lt 1 ]; then
     exit 1
 fi
 
+write_index_backup=$(./bin/console keyvalue:load:write)
```

Comment:
Yes. That was the intention to keep the name of the current read index so that if the new index is empty we will switch back to it. 

The function that I used `./bin/console keyvalue:load:write` was pointed out to me by @nlisgo when I asked how do we keep the current write index because the script switches the write index with the new one [here](https://github.com/elifesciences/search/blob/2d2080b4b7dde028c93bfef4c8f99818a431e382/bin/reindex#L20). 

I assume the `./bin/console index:write` command will return the current write index. So based on that assumption, I'll test if that works and make the changes.

Reply:
Ok. I've changed the command to `./bin/console index:read` because from my understanding the old index from read and write are the same (I've checked with @scottaubrey as well) so that should fix it.